scripts/chassismodulesshow

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python
+"""
+    Script to show chassis modules status.
+"""
+from __future__ import print_function
+
+import argparse
+
+from tabulate import tabulate
+from swsssdk import ConfigDBConnector
+from swsssdk import SonicV2Connector
+from natsort import natsorted
+
+
+header = ['Name', 'Description', 'Slot', 'Oper-Status', 'Admin-Status']
```

Comment:
admin@sonic:~$ show platform chassis-modules
         Name        Description    Slot    Oper-Status    Admin-Status
-------------  -----------------  ------  -------------  --------------
   LINE-CARD1  imm36-400g-qsfpdd       1         Online            down
   LINE-CARD2  imm36-400g-qsfpdd       2         Online              up

Can you update on the  max length of description field?
Also, get_description() call should pass the max so that platform doesn't exceed when filling the description data/string

Reply:
This is vendor specific and is dynamic in python code. The returned string will be put into a dictionary and then the database.